recipes/ncbi_refseq_accession_offsets/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ncbi_refseq_accession_offsets" %}
+{% set version = "2021.11.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/64/84/6e71ac3bd8aa091d6fbea545cda44e14bac0347841ad4085983bf891c9e5/{{ name }}-{{ version }}-py3-none-any.whl"
```

Comment:
Is there a possibility of switching to a source distribution? We generally don't prefer installations from wheels.